build-config

```diff
@@ -89,13 +91,25 @@ if [ "${FORKURL}" != "https://github.com/RackHD/on-core.git" ]; then
   fi
 fi
 
-if [ ! -z "${COMMIT}" ]; then
+IFS=', ' read -r -a FORKURL_ARRAY <<< "${FORKURL}"
+IFS=', ' read -r -a COMMIT_ARRAY <<< "${COMMIT}"
+if [ "${#FORKURL_ARRAY[@]}" == 2 ]; then
   if [ -e "../build/package.json" ]; then
     if [ "${GIT_REFSPEC}" != "master" ]; then
-      sed -i "s|"git+${FORKURL}#${GIT_REFSPEC}"|"git+${FORKURL}#${COMMIT}"|" ../build/package.json
+      sed -i "s|"git+${FORKURL_ARRAY[0]}#${GIT_REFSPEC}"|"git+${FORKURL_ARRAY[0]}#${COMMIT_ARRAY[0]}"|" ../build/package.json
+      sed -i "s|"git+${FORKURL_ARRAY[1]}#${GIT_REFSPEC}"|"git+${FORKURL_ARRAY[1]}#${COMMIT_ARRAY[1]}"|" ../build/package.json
     else
-      sed -i "s|"git+${FORKURL}"|"git+${FORKURL}#${COMMIT}"|" ../build/package.json
+      sed -i "s|"git+${FORKURL_ARRAY[0]}"|"git+${FORKURL_ARRAY[0]}#${COMMIT_ARRAY[0]}"|" ../build/package.json
+      sed -i "s|"git+${FORKURL_ARRAY[1]}"|"git+${FORKURL_ARRAY[1]}#${COMMIT_ARRAY[1]}"|" ../build/package.json
     fi
   fi
+elif [ ! -z "${COMMIT}" ]; then
+    if [ -e "../build/package.json" ]; then
+      if [ "${GIT_REFSPEC}" != "master" ]; then
+        sed -i "s|"git+${FORKURL}#${GIT_REFSPEC}"|"git+${FORKURL}#${COMMIT}"|" ../build/package.json
+      else
+        sed -i "s|"git+${FORKURL}"|"git+${FORKURL}#${COMMIT}"|" ../build/package.json
+      fi
+    fi
```

Comment:
When on-core and on tasks both exist. We must make sure  “on-http and on-taskgraph unit-test triggered by on-core” get the correct on-tasks pr branch, so FORKURL and COMMITS will be a list in such situation.